index.html

```diff
@@ -0,0 +1,13 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="/style.css">
+    <title>Document</title>
+</head>
+<body>
+    <h1 class="hello">Hello Microverse!</h1>
```

Comment:
- [ OPTIONAL] I can see that you call your CSS class in the `h1` tag which is not bad but for being more semantic in your code it could be better if you use the `div` tag and then give it a class name, like `div class="hello",..then you pass your   `h1` tag  :wink: 

I suggest you to have a look on this [Documentation](https://www.w3schools.com/cssref/sel_class.php#:~:text=To%20select%20elements%20with%20a,be%20affected%20by%20a%20class.) on how to do so

Reply:
Thanks @Alfredbis29 for your kind suggestion I will try to be more semantic in the future projects.